ahussain23V3.md

```diff
@@ -0,0 +1,28 @@
+Project Name:
+Smart Door Lock
+
+Members:
+@alihussain23
+@mehdihussain1786
+
+Summary
+What are you going to build? What does it do? Why are you excited to build it?
+My plan is to build an external smart door lock that can be controlled using a smartphone app. It will have a main base housing all the components, as well as a secondary base where the locking pins will mesh into. The main base will sit on the door and the secondary base will sit on the door frame, so when the locking pins extend outward, the door won't open.  As pre-made internal smart door locks can be quite expensive, and not many external ones exist with limited installation required, I believe this is an exciting project to pursue. I am also excited to work on with my little brother, as he always expressed interest in engineering and design like this but never knew where to start  
+
+Plan
+What steps are you going to take to build it? What tools are you going to use? What will you do first, second, third, etc.?
+First we plan on using a 3D printer to build the chassis as well as the locking pins for the lock. Prototypes will be built out of PLA to make sure everything is in order then the final model will be built of PETG for strength. The pins will be pushed by a servo and be controlled with an arduino which will be housed inside the chassis. Once all the parts are ordered/created, assembly should be pretty straightforward. Afterwards we plan to create the code for the arduino as well as develop the mobile app through the help of flutter and firebase. 
+
+Budget
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product         | Supplier/Link                                                                                        | Cost     |
+| --------------- | ---------------------------------------------------------------------------------------------------- | ------   |
+| Ender 3 S1 Pro  | https://www.microcenter.com/product/649022/creality-ender-3-s1-pro-3d-printer?ob=1&rf=Search+Results | $449     |
+| CR-10 V3.       | https://www.microcenter.com/product/633005/creality-cr-10-v3-3d-printer.                             | $349.    |
+| Sermoon D1.     | https://www.microcenter.com/product/637057/sermoon-d1-3d-printer?ob=1&rf=Search+Results.             | $449.    |
+| PETG Filament.  | https://www.microcenter.com/product/503745/inland-175mm-black-petg-3d-printer-filament-1kg-spool.    | $18.     |
+| Servo.          | https://www.amazon.com/Futaba-S3117-Micro-Hi-Torque-Servo/dp/B00EO8E9Y2                              | $30.     |
+| Total           |                                                                                                      | $400-500 |
```

Comment:
Don't forget to update the total! Btw, does it include shipping & taxes?

Reply:
Updated prices and total with tax included. Shipping is free for all products.